docs/introduction/faq-and-common-issues.md

```diff
@@ -148,3 +149,16 @@ See [Outgoing emails](https://github.com/djfarrelly/MailDev#outgoing-email) in t
 
 ## Can I see what is really happening in the Codeception acceptance tests when using Docker?
 Yes, you can! Check [the quick guide](/docs/introduction/running-acceptance-tests.md#how-to-watch-what-is-going-on-in-the-selenium-browser).
+
+## Why is there a faked PHP 7.2 platform in the Composer config?
+As a general rule, packages and libraries that depend on PHP 7.2 will work as expected even on PHP 7.3 (any higher 7.x version), but not vice versa.
+This is the maintainers of PHP focus on backward-compatibility (even if there were [some incompatible changes](https://www.php.net/manual/en/migration73.incompatible.php) introduced in PHP 7.3, in practice it doesn't cause issues).
+
+Using [the `config.platform.php` option](https://getcomposer.org/doc/06-config.md#platform) in `composer.json` allows us to force Composer to install such dependencies, that work for all supported versions of PHP by Shopsys Framework.
+These dependencies are locked during each release of SSFW so users that install it can download exact versions of all libraries and tools that were tested and proved working.
```

Comment:
We only use SSFW shortcut internally since I know. Also you used it a line above.
```suggestion
These dependencies are locked during each release of Shopsys Framework so users that install it can download exact versions of all libraries and tools that were tested and proved working.
```

Reply:
It's actually used across the docs, not only internally - see examples below. It's true that we use it quite sparingly. I didn't want to use the full name as there are more occurrences...

---

https://github.com/shopsys/shopsys/blob/6febd2cea2288a61cffadf224e9622dd4d683548/docs/cookbook/adding-a-new-administration-page.md#L19-L25
https://github.com/shopsys/shopsys/blob/6febd2cea2288a61cffadf224e9622dd4d683548/docs/kubernetes/how-to-deploy-ssfw-to-google-cloud-platform.md#L1
https://github.com/shopsys/shopsys/blob/6febd2cea2288a61cffadf224e9622dd4d683548/docs/model/model-rules.md#L7
https://github.com/shopsys/shopsys/blob/6febd2cea2288a61cffadf224e9622dd4d683548/docs/contributing/guidelines-for-pull-request.md#L78

It's even mentioned on our homepage https://www.shopsys.com/

>Shopsys Framework (**SSFW**) is a highly customizable ecommerce framework for businesses transitioning into tech-companies with their own software development team. It contains the most common B2C and B2B features for online stores, and its infrastructure is prepared for high scalability.

Reply:
So I guess it is right.

@TomasLudvik corrected me when I want using it in deprecation message https://github.com/shopsys/shopsys/pull/1345#discussion_r317007608 and that might be a different.